starknet_py/net/schemas/common.py

```diff
@@ -336,3 +337,27 @@ class StorageEntrySchema(Schema):
     def make_dataclass(self, data, **kwargs):
         # pylint: disable=no-self-use
         return StorageEntry(**data)
+
+
+class Revision(Enum):
+    """
+    Enum representing the revision of the specification to be used.
+    """
+
+    V0 = 0
+    V1 = 1
+
+
+class RevisionField(fields.Field):
```

Comment:
We need to implement the `_serialize_` method here as well, to ensure that the `dump` method works properly, for example with `TypedDataSchema`.

Reply:
Added 🫡